### ISSUE TITLE
Empty files trigger a timeout warning

### ISSUE THREAD

**Issue by kwridan** [nicklockwood/SwiftFormat#446]
### Overview

While running SwiftFormat on a project, I noticed that it displayed a timeout warning and an error:

```
$ swiftformat .
Running SwiftFormat...
warning: andOperator rule timed out.
error: Failed to format any files.
```

Running it on each file individually I discovered the culprit was an empty source file! This is a funny one, in a way glad SwiftFormat highlighted the fact there was an empty file which needed deleting - wanted to flag it incase it wasn't expected behaviour. Feel free to close this issue if it is expected.

### Reproduction Steps

Manually:

```
$ touch foo.swift
$ swiftformat foo.swit
```

Via unit tests: 

```swift
    func testEmptyFile() {
        let input = ""
        let output = ""
        XCTAssertEqual(try format(input, rules: FormatRules.all), output)
    }
```

Looks like it is occurring due to the timeout being zero in this situation when there are zero tokens.

https://github.com/nicklockwood/SwiftFormat/blob/master/Sources/SwiftFormat.swift#L405

A simple thing I could think of was to do an early return if there are no tokens to process

### Details

Running version 0.40.9

**Comment by nicklockwood** [comment]
@kwridan thanks for reporting this - I'll try to repro.

**Comment by nicklockwood** [comment]
@kwridan fixed in 0.40.10.

**Comment by kwridan** [comment]
Awesome, thanks @nicklockwood !
